content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/csi/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+layout: docs
+page_title: Direct secrets
```

Comment:
```suggestion
page_title: Use direct secrets in Kubernetes
```
Style correction: docs titles should not be features names. Instead, they should use direct language to indicate whether the information is conceptual or instructional.